modules/compute/virtual-machines/README.md

```diff
@@ -1026,6 +1026,8 @@ This section gives you an overview of all local-referenced module files (i.e., o
 
 | Reference | Type |
 | :-- | :-- |
```

Comment:
Blocking comment until singular names PR is merged

Reply:
I have merged your commits into the fail-fixing PR of mad proportions #3665